packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js

```diff
@@ -76,9 +78,7 @@ angular.module('App').controller(
         .then(({ modules, offers, user }) => {
           this.model.moduleTemplates = filter(modules, { branch: 'stable' });
           this.offers = compact(offers);
-          this.hostingUrl =
-            this.constants.urls.hosting[user.ovhSubsidiary] ||
-            this.constants.urls.hosting.FR;
+          this.hostingUrl = this.ORDER_URLS.orderHosting[user.ovhSubsidiary];
```

Comment:
Sure this isn't required anymore ?

Reply:
All subs hosting URLS are defined and we use `ng-if` for some display